app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_minicart.less

```diff
@@ -415,7 +414,7 @@
         margin-left: 13px;
 
         .block-minicart {
-            right: -15px;
+            right: -10px;
```

Comment:
you can remove this line as in line 113 it's set to -10px already

Reply:
Hi @Karlasa 
I have remove Duplicate code.